react/Popover.tsx

```diff
@@ -8,13 +8,17 @@ import TrapFocus from './components/TrapFocus'
 import Fade from './components/Animations/Fade'
 import OutsideClickHandler from './components/OutsideClickHandler'
 import { usePopoverState, usePopoverDispatch } from './PopoverContext'
+import useEventCallback from './modules/useEventCallback'
+import handleContainerStyle, { RestoreFn } from './modules/handleContainerStyle'
 
 type TransitionComponentType = 'fade'
 
 interface Props {
-  children: React.ReactNode
-  placement: Placement
   role?: string
+  placement: Placement
+  children: React.ReactNode
+  backdrop?: 'clickable' | 'none'
```

Comment:
isso aqui é a diferença entre aparecer um fundo e nao aparecer?

Reply:
se for, talvez `clickable` nao seja o ideal. `visible` seria mais claro, ou talvez `dim`

Reply:
O que é `dim`?
Tudo bem que o caso default é vc renderizar o backdrop com um `bg-base--inverted`, mas eu pensei em clickable pq o cara pode querer colcoar `background-color: transparent` e manter o uso de quando vc clicar nele fechar o `popover` sem interagir com os elementos do resto da página

Reply:
🤔🤔🤔